src/components/AssetDocumentsPanel/stories/AssetDocumentsPanel.stories.tsx

```diff
@@ -1,36 +1,51 @@
-import * as sdk from '@cognite/sdk';
+import { API } from '@cognite/sdk/dist/src/resources/api';
```

Comment:
This is not valid anymore in the SDK? The API-resource was deleted. Use `CogniteClient`